### PR TITLE
Serenity updates - November 2020 - Momo's news

### PR DESCRIPTION
**Pull request by NolwennP**
Left to do:
- [x] Add screenshots
- [x] Add doc links
- [x] Add "More" links

### CODE REVIEW

File: content/updates/2020-11/2-Import-and-export-your-reference-entity-records.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+::: meta-data type="New" features="Productivity" available="late November"
+
+Having a hard time initializing the records of your reference entities? We got you covered! You can now import an Excel or CSV file containing your records. We also make it possible for you to export them in flat files, once enriched!
+
+Discover this new feature in your `Import` and `Export` menus, under the following job names: 
+- `Reference entity record import in XSLX`,
+- `Reference entity record import in CSV`,
+- `Reference entity record export in XLSX`,
+- `Reference entity record export in CSV`.
+
```
Comment (LaureBro) [inline content/updates/2020-11/2-Import-and-export-your-reference-entity-records.md:16]:
```suggestion

This new feature can also help you bulk update your records or easily duplicate them.

```

Comment (NolwennP) [reply content/updates/2020-11/2-Import-and-export-your-reference-entity-records.md:16]: I'd suggest moving this sentence, after the picture.

File: content/updates/2020-11/2-Import-and-export-your-reference-entity-records.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+![TODO](../img/TODO.png)
+
```
Comment (NolwennP) [inline content/updates/2020-11/2-Import-and-export-your-reference-entity-records.md:18]:
```suggestion

This new feature can also help you bulk update your records or easily duplicate them.

```

File: content/updates/2020-11/2-Import-and-export-your-reference-entity-records.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+pim_announcement_audience:
+- EE
+---
+
+# Import and export your reference entity records
+::: meta-data type="New" features="Productivity" available="late November"
+
+Having a hard time initializing the records of your reference entities? We got you covered! You can now import an Excel or CSV file containing your records. We also make it possible for you to export them in flat files, once enriched!
```
Comment (LaureBro) [inline content/updates/2020-11/2-Import-and-export-your-reference-entity-records.md:9]: @NolwennP, I wonder if "flat files" speak to our users, for me it's a tech word. Should we use (again) XLSX and CSV file formats or say 
"You can now import and export reference entity records through Excel or CSV."

Comment (NolwennP) [reply content/updates/2020-11/2-Import-and-export-your-reference-entity-records.md:9]: You're right about the "flat files" expression. Not sure, it's well known.
But I cant use your suggestion as exporting is not answering the initialization scenario. I'd suggest:
```suggestion
Having a hard time initializing the records of your reference entities? We got you covered! You can now import an Excel or CSV file containing your records. We also make it possible for you to export them in Excel or CSV, once enriched. So neat!
```

File: content/updates/2020-11/6-Meet-the-new-yes-no-attribute.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+::: meta-data type="Improvement" features="Productivity" available="late November"
+
+The Yes/No attribute is now more powerful: you can easily see if your Yes/No attribute has no value yet. Pretty convenient and way less misleading than before!
+
+![New Yes/No attribute design](../img/new-yes-no-attribute-design.png)
+
+## Better filter with the Yes/No attribute
+::: meta-data type="Improvement" features="Productivity" available="late November"
+
+Two new operators are now available when filtering on Yes/No attributes in your product grid and rules engine: `is empty` and `is not empty`. Those new filters we help you gain in productivity and be even more efficient when looking for specific products and product models.
```
Comment (morganen) [inline content/updates/2020-11/6-Meet-the-new-yes-no-attribute.md:23]:
```suggestion
Two new operators are now available when filtering on Yes/No attributes in your product grid and rules engine: `is empty` and `is not empty`. Those new filters help you gain in productivity and be even more efficient when looking for specific products and product models.
```

File: content/updates/2020-11/6-Meet-the-new-yes-no-attribute.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+![New is empty and is not empty filters on the Yes/No attributes](../img/is-empty-is-not-empty-filters.png)
+
+The first one, `is empty`, allows you to search for products and product models that have Yes/No attribute with no value selected. The seconde one, `is not empty` allows you to search for the contrary.
```
Comment (morganen) [inline content/updates/2020-11/6-Meet-the-new-yes-no-attribute.md:27]:
```suggestion
The first one, `is empty`, allows you to search for products and product models that have Yes/No attribute with no value selected. The second one, `is not empty` allows you to search for the contrary.
```

Comment (morganen) [inline content/updates/2020-11/6-Meet-the-new-yes-no-attribute.md:27]:
```suggestion
The first one, `is empty`, allows you to search for products and product models that have Yes/No attribute with no value selected. The second one, `is not empty` allows you to search for the contrary.
```

File: content/updates/2020-11/6-Meet-the-new-yes-no-attribute.md (hunk 4 above)
Comment (morganen) [inline content/updates/2020-11/6-Meet-the-new-yes-no-attribute.md:23]:
```suggestion
Two new operators are now available when filtering on Yes/No attributes in your product grid and rules engine: `is empty` and `is not empty`. Those new filters help you gain in productivity and be even more efficient when looking for specific products and product models.
```

File: content/updates/2020-11/3-Create-private-views.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+pim_announcement_alt_img: "Create a private view"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Create private views
+::: meta-data type="New" features="Productivity" available="late November" link-to-doc="../articles/manage-your-views.html#create-a-view"
+
+Stay organised in your product thanks to private views! It's now possible to create private views, only visible by you, to help you focus on your own enrichment needs without polluting your teammates.
```
Comment (NolwennP) [inline content/updates/2020-11/3-Create-private-views.md:12]:
```suggestion
Stay organized in your product enrichment thanks to private views! It's now possible to create private views, only visible by you, to help you focus on your own enrichment needs without polluting your teammates. 
```

**Review by LaureBro — APPROVED** [review]

**Pull request merged**